balena.yml

```diff
@@ -0,0 +1,6 @@
+applicationEnvironmentVariables:
+  - CODE_WORKSPACE_PATH: "/usr/src/projects"
+  - CODE_SETTINGS_PATH: "/usr/src/settings"
+  - GIT_USER_NAME: "CHANGE-ME"
+  - GIT_USER_EMAIL: "CHANGE-ME"
+  - CLI_TOKEN: "CHANGE-ME"
```

Comment:
Can you change this ones to `""`?
I'm using `test -n` [here](https://github.com/balena-io-playground/balena-ide/blob/master/code/start.sh#L36) and [here](https://github.com/balena-io-playground/balena-ide/blob/master/code/start.sh#L51) so i think empty strings are better defaults as they won't trigger the logic. (not the best way of doing things I know haha)